tests/test_examples.py

```diff
@@ -144,66 +122,6 @@ def test_HELIOTRON_vac_results(HELIOTRON_vac):
     np.testing.assert_allclose(curr(np.linspace(0, 1, 20)), 0, atol=1e-8)
 
 
-@pytest.mark.regression
-@pytest.mark.solve
-def test_HELIOTRON_vac2_results(HELIOTRON_vac, HELIOTRON_vac2):
-    """Tests that the 2 methods for solving vacuum give the same results."""
-    eq1 = EquilibriaFamily.load(load_from=str(HELIOTRON_vac["desc_h5_path"]))[-1]
-    eq2 = EquilibriaFamily.load(load_from=str(HELIOTRON_vac2["desc_h5_path"]))[-1]
-    rho_err, theta_err = area_difference_desc(eq1, eq2)
-    np.testing.assert_allclose(rho_err[:, 4:], 0, atol=1e-2)
-    np.testing.assert_allclose(theta_err, 0, atol=1e-4)
-    curr1 = eq1.get_profile("current")
-    curr2 = eq2.get_profile("current")
-    iota1 = eq1.get_profile("iota")
-    iota2 = eq2.get_profile("iota")
-    np.testing.assert_allclose(curr1(np.linspace(0, 1, 20)), 0, atol=1e-8)
-    np.testing.assert_allclose(curr2(np.linspace(0, 1, 20)), 0, atol=1e-8)
-    np.testing.assert_allclose(iota1.params, iota2.params, rtol=1e-1, atol=1e-1)
-
-
-@pytest.mark.regression
-@pytest.mark.solve
-def test_force_balance_grids():
```

Comment:
this has been a do-nothing test for a while, note the comment that the grids aren't even different anymore